containers/ecr-viewer/src/app/view-data/components/AccordionLabResults.tsx

```diff
@@ -43,7 +46,7 @@ export const AccordionLabResults: React.FC<AccordionLabResultsProps> = ({
           content: content,
           expanded: collapsedByDefault,
           id: title,
-          headingLevel: "h5",
+          headingLevel,
           className: `acc_item_${organizationId}`,
```

Comment:
Seems like the side nav is picking up the labs [because it's not an h5 anymore](https://github.com/CDCgov/phdi/blob/a411f8365535f4427a278766d60f4f05798d1154/containers/ecr-viewer/src/app/view-data/components/SideNav.tsx#L35). If you just add `side-nav-ignore` to this class name it should be resolved since  no lab results should be in the sidenav anyways.
<img width="1495" alt="image" src="https://github.com/user-attachments/assets/b5cdc8e2-a6ae-47ab-86b1-a2467f003438">
```suggestion
          className: `acc_item_${organizationId} side-nav-ignore`,
```